v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/v1.kt

```diff
@@ -51,6 +51,18 @@ internal suspend inline fun <reified T> ExtendableApiClient.getInternal(url: Url
     }
 }
 
+@OptIn(Twitter4ktInternalAPI::class)
+internal suspend inline fun <reified T> ExtendableApiClient.getInternalListResponse(url: Url): ApiResult<List<T>> {
```

Comment:
Do we need this?

I think `getInternal` can handle List response too.

Reply:
> Do we need this?
> 
> I think `getInternal` can handle List response too.

I wasn't able to get it to do that with my experimentation.

Reply:
Ok, I'll revisit this later, as I think we don't need to encode to  `JsonObject` in `getInternal` (I have no idea why I wrote this.)

For now, you can keep this implementation 👍